test/request.js

```diff
@@ -51,6 +51,27 @@ describe('request', () => {
         assert.equal(httpStub.request.callCount, 1);
     });
 
+    it('should use pathname as path if none specified', () => {
+        const requestStub = new RequestStub();
```

Comment:
its already like this, but it would be better if you can move these two lines to beforeEach to avoid duplication